### ISSUE TITLE
Pre-Compiled exe is trojan

### ISSUE THREAD

**Issue by Diddlyh** [elastic/PPLGuard#2]
Don't download or execute this file or source code! See for your self:

https://www.joesandbox.com/analysis/1275635

https://www.joesandbox.com/analysis/1275635/0/html
https://www.joesandbox.com/analysis/1275635/1/html
https://www.joesandbox.com/analysis/1275635/2/html

Don't see the file doing what it was supposed to do. Instead it behaves in various way very maliciously.

**Comment by gabriellandau** [comment]
No.  It's an exploit, not a trojan.  It contains no hidden functionality.  See the [README](https://github.com/elastic/PPLGuard/blob/main/README.md#pplguard).

> To apply these mitigations, PPLGuard exploits an unpatched Windows local privilege escalation vulnerability to execute code with WinTcb Protected Processs (PP) privileges.

It's [based off PPLDump](https://github.com/elastic/PPLGuard/blob/main/README.md#credits), which explains the FPs.

> [@itm4n](https://twitter.com/itm4n) This is is 95% @itm4n's [PPLDump](https://github.com/itm4n/PPLdump) code, with the exploit payload replaced.
